### PR TITLE
Added Igor's list of publications

### PR DESCRIPTION
**Pull request by osg-cat**

### CODE REVIEW

File: _about/publications.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
 
+This page lists publications and presentations to which OSG staff contributed.
+
+# Publications
+
+## 2021
+
+## 2020
+
+*   J. Dost et al,
```
Comment (djw8605) [inline _about/publications.md:14]: What an awful author list.

File: _about/publications.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
...
+    “A Lightweight Door into Non-Grid Sites,”
+    EPJ Web of Conferences 245, 07005 (2020).
+    https://doi.org/10.1051/epjconf/202024507005
+
+*   E. Fajardo, et al,
```
Comment (djw8605) [inline _about/publications.md:19]: Same, at least put all the authors...

**Comment by jstathas** [comment]
Tim,


Do you want to address Derek's suggestions for a more complete author list instead of et al before I merge?


Janet


Janet Stathas
PATh Project Manager
Morgridge Institute for Research

Email: ***@***.******@***.***>
Cell:  608.345.9605

https://morgridge.org<https://morgridge.org/>
https://path-cc.io/


________________________________

**Comment by osg-cat** [comment]
@djw8605, while I agree in principle, we need to get a page up and fleshed out a bit ASAP. Clean up can come later. So I would like to go ahead with this initial commit. The next step is going to be to ask you (and others) to interleave your publications and presentations into this page. If you want to take time to clean things up a bit, too, go right ahead!

**Pull request merged**